src/main/java/org/folio/edge/rtac/MainVerticle.java

```diff
@@ -1,14 +1,12 @@
 package org.folio.edge.rtac;
 
-import org.folio.edge.core.EdgeVerticle;
-import org.folio.edge.core.utils.Mappers;
-import org.folio.edge.rtac.utils.RtacOkapiClientFactory;
-
 import com.fasterxml.jackson.annotation.JsonInclude;
-
 import io.vertx.core.http.HttpMethod;
 import io.vertx.ext.web.Router;
 import io.vertx.ext.web.handler.BodyHandler;
+import org.folio.edge.core.EdgeVerticle;
+import org.folio.edge.core.utils.Mappers;
+import org.folio.edge.rtac.utils.RtacOkapiClientFactory;
```

Comment:
I just used code formatting capabilities provided by ItelliJ Idea Community. I'll compare it with master branch to avoid unneeded code modifications. Thank you.

Reply:
In mod-circulation, we had lots of false changes due to the major IDEs having different default configurations for this. Unofficially (because FOLIO hasn't agreed on it as a standard) I chose to adopt the Eclipse ordering (even though I use IntelliJ). How to configure IntelliJ to support this is described in [FOLIO-1667](https://issues.folio.org/browse/FOLIO-1667)

I don't know if edge-rtac uses the same approach. If the pre-pull request ordering was based upon eclipse, this configuration should resolve this for future work.

Reply:
Thanks for sharing the link on configuration. Anyway that's a good point to do visual checks for possible import rearrangement.